src/settings.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>TokyoDrift: Settings</title>
+
+  </head>
+  <body>
+    <h1 style="text-align: center;">Settings</h1>
+    <hr />
+    <table>
+        <tr>
+            <td>Work period ("Pomodoro") length</td>
+            <td><input type="number" value="25" min="1" max="1440" /> minutes</td>
+        </tr>
+        <tr>
+            <td>Short break length</td>
+            <td><input type="number" value="5" min="1" max="1440" /> minutes</td>
+        </tr>
+        <tr>
+            <td>Long break length</td>
+            <td><input type="number" value="20" min="1" max="1440" /> minutes</td>
+        </tr>
+        <tr>
+            <td>Number of work periods per session</td>
+            <td><input type="number" value="4" min="2" max="20" /> work periods</td>
```

Comment:
One thing that could be done is to give an id to each input. This would help when we want to get the result in JS and it is also a good practice that can help when testing.